pkg/cmd/provisioning/aws/create_identity_provider_test.go

```diff
@@ -25,7 +25,7 @@ import (
 
 const (
 	testInfraName     = "test-infra-name"
-	testRegionName    = "test-region"
+	testRegionName    = "us-west-2"
```

Comment:
This needs to be a real region in the tests since we use the region to find the DNS suffix.